src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -1,6 +1,11 @@
 # We don't properly capitalize SCM in the class name since CreateJob is doing `CLASS_NAME.to_s.camelize.safe_constantize`
 class ReportToScmJob < CreateJob
-  ALLOWED_EVENTS = ['Event::BuildFail', 'Event::BuildSuccess'].freeze
+  ALLOWED_EVENTS = [
+    'Event::BuildBroken',
+    'Event::BuildFail',
+    'Event::BuildSuccess',
```

Comment:
Note that a build can switch from success to fail and back to unchanged. So if you reported fail, you need to look at BuildUnchanged too

Reply:
So `Event::BuildUnchanged` is also to be reported as `success`... this is weird, but whatever, this is how it is.

https://github.com/openSUSE/open-build-service/blob/af9297d8d8fb79e14951f657a792474c53179e67/src/api/app/models/event/build_unchanged.rb#L4

Reply:
Don't shoot the messenger :)